### PR TITLE
Coworking app version 1

### PR DESCRIPTION
**Pull request by azamat-dev043**
first pull request

### CODE REVIEW

**Comment by azamat-dev043** [comment]
first comment

File: src/Main.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        Customer customer = new Customer(reservations);
+
+        Scanner scanner = new Scanner(System.in);
+        int choice;
+
+
+        do {
+            System.out.println("\n--- Welcome to Azamat's Coworking space ---");
+            System.out.println("\n--------------Main Menu--------------------");
+            System.out.println("1. Admin Login");
```
Comment (elnurmes) [inline src/Main.java]: use java text blocks here: https://www.baeldung.com/java-text-blocks

**Review by elnurmes — CHANGES_REQUESTED** [review]
Hi, good job in general, pls fix the comment I added

File: src/Main.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        Customer customer = new Customer(reservations);
+
+        Scanner scanner = new Scanner(System.in);
+        int choice;
+
+
+        do {
+            System.out.println("""
+                    \n--- Welcome to Azamat's Coworking space ---
+                    --------------Main Menu----------------------
```
Comment (elnurmes) [inline src/Main.java]: try to use java text blocks: https://www.baeldung.com/java-text-blocks

File: src/Main.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                        case 3:
+                            admin.viewReservations(reservations);
+                            break;
+                    }
+                    break;
+
+                case 2:
+                    System.out.println("/n---Customer Menu---");
+                    System.out.println("1. Browse available spaces");
+                    System.out.println("2. Make a reservation");
```
Comment (elnurmes) [inline src/Main.java]: same here

**Review by elnurmes — CHANGES_REQUESTED** [review]
Hi! Good job in general, added small comments, pls fix them

File: src/Main.java (hunk 2 above)
Comment (azamat-dev043) [reply src/Main.java]: solved

**Review by elnurmes — APPROVED** [review]
you can merge it

**Pull request merged**